src/Service/ApiService.php

```diff
@@ -405,7 +405,7 @@ public function getFilteredApiOrder($api, $transactionId)
                 'currency' => $payment->amount->currency
             ];
             $order['availableRefundAmount'] = $payment->amountRemaining ?: $amountRemaining;
-            $order['details'] = $payment->details;
+            $order['details'] = $payment->details ?: new \stdClass();
```

Comment:
can we have for order details our own created object ?  How much of an effort it should be?

Reply:
but why? we dont know what i can have because each payment have different details and some of them dont have any. its possible but i m not sure we need it because we only use it in js side